RecoTracker/TkTrackingRegions/src/OuterHitPhiPrediction.h

```diff
@@ -4,32 +4,40 @@
 /** predicts phi range at a given radius r */
 
 #include "RecoTracker/TkMSParametrization/interface/PixelRecoRange.h"
-#include "RecoTracker/TkTrackingRegions/interface/TkTrackingRegionsMargin.h"
 
 #include "FWCore/Utilities/interface/Visibility.h"
 
+#include <cassert>
+
 class dso_internal OuterHitPhiPrediction {
 public:
 
-  typedef PixelRecoRange<float> Range;
-  typedef TkTrackingRegionsMargin<float> Margin;
+  using Range= PixelRecoRange<float>;
 
   OuterHitPhiPrediction( 
       const Range & phiAtVertex, 
       const Range & curvature, 
-      float originRBound, 
-      const Margin & tolerance = Margin(0,0))
+      float originRBound) 
     : thePhiAtVertex(phiAtVertex), theCurvature(curvature),
-      theOriginRBound (originRBound), theTolerance(tolerance) { } 
+      theOriginRBound (originRBound) {
+       assert(theCurvature.max()>0);
```

Comment:
```80X_dataRun2_HLT_v12``` works also!
(BTW, note that in all these tests I add the L1T menu explicitly via toGet)